pom.xml

```diff
@@ -929,6 +929,15 @@ under the License.
 								<excludedGroups>org.apache.flink.testutils.junit.FailsOnJava11</excludedGroups>
 							</configuration>
 						</plugin>
+						<plugin>
+							<groupId>org.apache.maven.plugins</groupId>
+							<artifactId>maven-javadoc-plugin</artifactId>
+							<configuration>
+								<additionalJOptions>
+									<additionalJOption>--add-exports=java.base/sun.net.util=ALL-UNNAMED</additionalJOption>
```

Comment:
Could you quickly explain what this does?

Reply:
It makes one of the internal jdk packages accessible at runtime to my understanding.

We need the same thing for the `maven-compiler-plugin` (although curiously not in dist....).